core/src/impl/Kokkos_Atomic_Generic_Secondary.hpp

```diff
@@ -72,5 +72,15 @@ KOKKOS_INLINE_FUNCTION void atomic_sub(volatile T* const dest, const T val) {
   (void)atomic_fetch_sub(dest, val);
 }
 
+template <typename T>
+KOKKOS_INLINE_FUNCTION void atomic_mul(volatile T* const dest, const T val) {
+  (void)atomic_fetch_mul(dest, val);
+}
+
+template <typename T>
+KOKKOS_INLINE_FUNCTION void atomic_div(volatile T* const dest, const T val) {
+  (void)atomic_fetch_div(dest, val);
+}
+
```

Comment:
What does this have to do with OMPTarget?

Reply:
nothing which is why its also in this PR: #3162